xsd/netex_part_2/part2_vehicleService/netex_vehicleRechargingPlan_support.xsd

```diff
@@ -0,0 +1,123 @@
+Business and industry, Transport, Air transport , Ports and maritime transport, Public transport,
+Rail transport, Roads and Road transport
+</Category>
+					<Project>CEN TC278 WG3 SG9.</Project>
+				</Subject>
+				<Title>NeTEx RECHARGING PLAN identifier  types.</Title>
+				<Type>Standard</Type>
+			</Metadata>
+		</xsd:appinfo>
+		<xsd:documentation>NeTEx: RECHARGING PLAN identifier types.</xsd:documentation>
+	</xsd:annotation>
+	<!-- ==== RECHARGING PLAN===================================================== -->
+	<xsd:simpleType name="RechargingPlanIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a RECHARGING PLAN.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="ObjectIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="RechargingPlanRef" type="RechargingPlanRefStructure" substitutionGroup="VersionOfObjectRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a RECHARGING PLAN. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="RechargingPlanRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for Reference to a RECHARGING PLAN.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="VersionOfObjectRefStructure">
+				<xsd:attribute name="ref" type="RechargingPlanIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a RECHARGING PLAN.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<!-- ===== RECHARGING STEP ==================================================== -->
+	<xsd:simpleType name="RechargingStepIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a RECHARGING STEP.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="ObjectIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="RechargingStepRef" type="RechargingStepRefStructure" substitutionGroup="VersionOfObjectRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a RECHARGING STEP. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="RechargingStepRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for Reference to a RECHARGING STEP.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="VersionOfObjectRefStructure">
+				<xsd:attribute name="ref" type="RechargingStepIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a RECHARGING STEP.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="RechargingProcessEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed value for RECHARGING PROCESS.</xsd:documentation>
```

Comment:
```suggestion
			<xsd:documentation>Allowed value for RECHARGING PROCESS. +v2.0</xsd:documentation>
```

Reply:
For me it isn't completely clear where signalling "+2.0" is needed and where not...